doc/Configuration.rst

```diff
@@ -95,6 +95,15 @@ One common use case for this feature is to increase the maximum heap space avail
 
     Specifying 32GB of memory available to ImageJ ecosystem routines in the JVM.
 
+Using the SciJava REPL
+--------------------------------
+
+You can use the SciJava REPL to interactively run SciJava code. This makes it possible to do things like paste existing SciJava scripts into the REPL. More information on scripting in SciJava can be found `here <https://imagej.net/scripting/>`_.
```

Comment:
So I played a little with the Script REPL, and I couldn't get anything beyond `2+4` to work. I was trying to print `Hello World`, but I couldn't even figure out how to do that.

So, as a result, I looked here, and I checked out imagej.net/scripting. As someone who is not intimately familiar with that page, I wasn't super sure where to go. I did find [the ScriptInterpreter page](https://imagej.net/scripting/interpreter) from that page, which *appears to me to be what this is*, but I tried typing `:help` and I didn't get anything.

Then, I typed `help`, and my napari froze. I went back to my terminal, and I saw this:

```bash
(napari-imagej-dev) ➜  napari-imagej git:(scijava-repl) napari
hello
[java.lang.Enum.toString] hello[java.lang.Enum.toString] 
Hello World
[java.lang.Enum.toString] Hello World[java.lang.Enum.toString] 
Hello World
[java.lang.Enum.toString] Hello World[java.lang.Enum.toString] 
Hello World
[java.lang.Enum.toString] Hello World[java.lang.Enum.toString] 
Hello World
[java.lang.Enum.toString] Hello World[java.lang.Enum.toString] 

Welcome to Python 2.7!  This is the online help utility.

If this is your first time using Python, you should definitely check out
the tutorial on the Internet at http://docs.python.org/2.7/tutorial/.

Enter the name of any module, keyword, or topic to get help on writing
Python programs and using Python modules.  To quit this help utility and
return to the interpreter, just type "quit".

To get a list of available modules, keywords, or topics, type "modules",
"keywords", or "topics".  Each module also comes with a one-line summary
of what it does; to list the modules whose summaries contain a given word
such as "spam", type "modules spam".

help> [java.lang.Enum.toString] 
Welcome to Python 2.7!  This is the online help utility.

If this is your first time using Python, you should definitely check out
the tutorial on the Internet at http://docs.python.org/2.7/tutorial/.

Enter the name of any module, keyword, or topic to get help on writing
Python programs and using Python modules.  To quit this help utility and
return to the interpreter, just type "quit".

To get a list of available modules, keywords, or topics, type "modules",
"keywords", or "topics".  Each module also comes with a one-line summary
of what it does; to list the modules whose summaries contain a given word
such as "spam", type "modules spam".

help> 
```
So, long story short, it looks like the output is not being captured by the TextEdit, and also I'm wondering if we can link to documentation that is more specific to the SciJava REPL.

To solve the output not being captured by the TextEdit, is there an option in SciJava for this @ctrueden?

Reply:
We now have it where the input and output is being displayed identically to the SciJava ScriptREPL - any further problems, including this one, should be fixed upstream.